test/Feature/QuerySpec.hs

```diff
@@ -799,8 +799,31 @@ spec = do
         get "/items_with_different_col_types?time_data=in." `shouldRespondWith`
           [json| [] |] { matchHeaders = [matchContentTypeJson] }
 
+    it "returns an empty result for notin and not.in when no value is present" $ do
```

Comment:
Shouldn't "not.in nothing" actually match everything? I'm thinking of it as being the complement of the empty set. It's a weird situation though, glad there's a test for it, whatever we decide.